src/server/server.ts

```diff
@@ -62,8 +62,7 @@ app.use(
     limit: "1mb",
     type: (req) => {
       for (const allowedType of allowedBodyContentTypes) {
-        // @ts-ignore - TODO: type definition doesn't include `is()`
-        if (req.is(allowedType)) {
+        if ((req as express.Request).is(allowedType)) {
```

Comment:
Jeg vet ikke hvorfor denne er typet som `http.IncomingMessage` og ikke `express.Request`, men jeg kan heller ikke se for meg at den noen gang skal være noe annet. Tanker?

Reply:
Ser at IncomingMessage har en headers-array. Burde ikkje verdien ligge der? Kan dette feile fordi typen er forskjellig fra det du caster?

Reply:
Dersom det er noe som ikke har `is` så kan det jo potensielt feile.
Ja den har et headers array, men `content-type` formatet er litt mer komplekst enn at vi bare kan sjekke at den er lik `allowedType`.

Så dersom vi skal bruke den så må vi nesten ta i bruk et bibliotek for at vi skal være sikre på at det ikke skal brekke.
Dette brakk jo pga noen sendte inn `charset` feks.